google/cloud/accessapproval/access_approval_client.h

```diff
@@ -20,440 +20,14 @@
-  /// @param request
-  /// @googleapis_link{google::cloud::accessapproval::v1::DeleteAccessApprovalSettingsMessage,google/cloud/accessapproval/v1/accessapproval.proto#L686}
-  /// @param opts Optional. Override the class-level options, such as retry and
-  ///     backoff policies.
-  ///
-  /// [google.cloud.accessapproval.v1.DeleteAccessApprovalSettingsMessage]:
-  /// @googleapis_reference_link{google/cloud/accessapproval/v1/accessapproval.proto#L686}
-  ///
-  Status DeleteAccessApprovalSettings(
-      google::cloud::accessapproval::v1::
-          DeleteAccessApprovalSettingsMessage const& request,
-      Options opts = {});
-
-  ///
-  /// Retrieves the service account that is used by Access Approval to access
-  /// KMS keys for signing approved approval requests.
-  ///
-  /// @param name  Name of the AccessApprovalServiceAccount to retrieve.
-  /// @param opts Optional. Override the class-level options, such as retry and
-  ///     backoff policies.
-  /// @return
-  /// @googleapis_link{google::cloud::accessapproval::v1::AccessApprovalServiceAccount,google/cloud/accessapproval/v1/accessapproval.proto#L562}
-  ///
-  /// [google.cloud.accessapproval.v1.GetAccessApprovalServiceAccountMessage]:
-  /// @googleapis_reference_link{google/cloud/accessapproval/v1/accessapproval.proto#L694}
-  /// [google.cloud.accessapproval.v1.AccessApprovalServiceAccount]:
-  /// @googleapis_reference_link{google/cloud/accessapproval/v1/accessapproval.proto#L562}
-  ///
-  StatusOr<google::cloud::accessapproval::v1::AccessApprovalServiceAccount>
-  GetAccessApprovalServiceAccount(std::string const& name, Options opts = {});
-
-  ///
-  /// Retrieves the service account that is used by Access Approval to access
-  /// KMS keys for signing approved approval requests.
-  ///
-  /// @param request
-  /// @googleapis_link{google::cloud::accessapproval::v1::GetAccessApprovalServiceAccountMessage,google/cloud/accessapproval/v1/accessapproval.proto#L694}
-  /// @param opts Optional. Override the class-level options, such as retry and
-  ///     backoff policies.
-  /// @return
-  /// @googleapis_link{google::cloud::accessapproval::v1::AccessApprovalServiceAccount,google/cloud/accessapproval/v1/accessapproval.proto#L562}
-  ///
-  /// [google.cloud.accessapproval.v1.GetAccessApprovalServiceAccountMessage]:
-  /// @googleapis_reference_link{google/cloud/accessapproval/v1/accessapproval.proto#L694}
-  /// [google.cloud.accessapproval.v1.AccessApprovalServiceAccount]:
-  /// @googleapis_reference_link{google/cloud/accessapproval/v1/accessapproval.proto#L562}
-  ///
-  StatusOr<google::cloud::accessapproval::v1::AccessApprovalServiceAccount>
-  GetAccessApprovalServiceAccount(
-      google::cloud::accessapproval::v1::
-          GetAccessApprovalServiceAccountMessage const& request,
-      Options opts = {});
-
- private:
-  std::shared_ptr<AccessApprovalConnection> connection_;
-  Options options_;
-};
+using ::google::cloud::accessapproval_v1::AccessApprovalClient;
```

Comment:
Should the forwarding headers say anything about preferring the versioned names, like an `@deprecated` or an `@note`?  (Sorry I didn't ask this in the generator PR.)

Reply:
I think so. I opened #10466. I'll rebase this one (and probably clean up the quickstart commits) after it goes through.

Reply:
SGTM.